helmfile.d/lists/images.yaml

```diff
@@ -15,9 +15,11 @@ images:
       helm: xpkg.crossplane.io/crossplane-contrib/provider-helm:v1.0.0
     functions:
       auto-ready: xpkg.crossplane.io/crossplane-contrib/function-auto-ready:v0.5.0
-      capability: harbor.long-running.dev-ck8s.com/welkin/function-capability:v0.2.0
+      capability: harbor.long-running.dev-ck8s.com/welkin/function-capability:fieldpath
       go-templating: xpkg.crossplane.io/crossplane-contrib/function-go-templating:v0.11.0
       patch-and-transform: xpkg.crossplane.io/crossplane-contrib/function-patch-and-transform:v0.9.0
+    configurations:
+      module-node-local-dns: harbor.long-running.dev-ck8s.com/welkin/module-node-local-dns:fieldpath
```

Comment:
TODO: Update when https://github.com/elastisys/module-node-local-dns/pull/10 is released

Reply:
https://github.com/elastisys/compliantkubernetes-apps/pull/2780/commits/c6cd19a33b3dda62bea3026833420baedf96f936